polling_stations/apps/councils/models.py

```diff
@@ -9,7 +9,7 @@ class Council(models.Model):
     website = models.URLField(blank=True)
     postcode = models.CharField(blank=True, null=True, max_length=100)
     address = models.TextField(blank=True, null=True)
-    area = models.MultiPolygonField(null=True, blank=True, geography=True, srid=4326)
+    area = models.MultiPolygonField(null=True, blank=True, srid=4326)
```

Comment:
I've got this weird nagging feeling there is code somewhere that expects this, but I can't find it for the life of me, so I reckon just crack on with it..

Reply:
I've had quite a good scour through the code base and can't see anything. It also doesn't make loads of sense that we should need the geography type for an application that applies to the UK. But I'll keep it in mind if anything weird crops up.